CharacterCreation/Class-Specific Documentation/Engineer Processes.txt

```diff
@@ -674,8 +674,8 @@ Plasma induction core explodes violently when destroyed. The titan comes with a
 7.62 repeater that has a 2 chance to miss and deals 55 damage and has an armor 
 piercing level of 4 and an auto fire rate of 5. The repeater is a Heavy 
 suppression weapon fed from a 150 round drum. May also make melee attacks with 
-its offhand with a +3 bonus. Deals 160 damage, kinda like being punched by a 
-backhoe.
+its offhand with a Melee Miss Chance of 3. Deals 160 damage, kinda like being 
```

Comment:
Melee doesn't have a 'miss chance' anymore. There isn't a 'melee miss chance' because the attacker rolls a melee attack vs the opponent's guard DC. 
https://github.com/bleehu/Compound_X/blob/37f9e8d2537dc1844a7e710c06b2375f2572d812/01_01_Basic_Rules.txt#L755